anomaly_detector/fact_store/app.py

```diff
@@ -74,17 +81,5 @@ def feedback():
     return ""
 
 
-@app.route("/api/anomaly_event", methods=["POST"])
```

Comment:
why do you remove this in this PR? it's not related to the counter.

Reply:
We are no longer recording events as its data duplication. We have single table which records just the feedback and the log line which is not an anomaly.